subprojects/wrapper/src/main/java/org/gradle/wrapper/Download.java

```diff
@@ -35,6 +35,8 @@
 
     private static final int BUFFER_SIZE = 10 * 1024;
     private static final int PROGRESS_CHUNK = 1024 * 1024;
+    private static final int CONNECTION_TIMEOUT_SECONDS = 60 * 1000;
+    private static final int READ_TIMEOUT_SECONDS = 60 * 1000;
```

Comment:
Trying this out locally, I wonder if we should go with a shorter timeout (say, 10 seconds) since this is a timeout for connecting/reading _some_ data and not downloading the whole thing.  WDYT?